### ISSUE TITLE
Use an aria-labelledby attribute to connect the tooltip to the slider component

### ISSUE THREAD

**Issue by Veyfeyken** [digipolisantwerp/acpaas-ui_react#212]
## Problem

The tooltips in "Bar: slider with tooltips" are not announced by screenreaders. Only the value of the aria-valuenow attribute is read aloud. 

```html
<div class="m-range-slider__handle" tabindex="0" role="slider" aria-orientation="horizontal" aria-valuemax="100" aria-valuemin="0" aria-valuenow="50" style="left: 436px;">
 <div class="m-range-slider__tooltip a-tooltip a-tooltip--primary a-tooltip--top">
  <p>50 year</p>
 </div>
</div>
```

## Suggestion

Connect to tooltip to the slide with an aria-labelledby attribute:

```html
<div class="m-range-slider__handle" tabindex="0" role="slider" aria-labelledby="tooltip" aria-orientation="horizontal" aria-valuemax="100" aria-valuemin="0" aria-valuenow="50" style="left: 436px;">
 <div class="m-range-slider__tooltip a-tooltip a-tooltip--primary a-tooltip--top">
  <p id="tooltip">50 year</p>
 </div>
</div>
```

## Articles and demo
* [ARIA design pattern: slider](https://www.w3.org/TR/wai-aria-practices-1.2/#slider)
* [Example](https://www.w3.org/TR/wai-aria-practices-1.2/examples/slider/slider-1.html)

**Comment by Veyfeyken** [comment]
An alternative approach would be to use the aria-valuetext attribute to include "years" as demonstrated in the multi-thumb example:

* [ARIA spec: aria-valuetext attribute](https://www.w3.org/TR/wai-aria-1.2/#aria-valuetext)
* [ARIA design pattern for multi-thumb slider example](https://www.w3.org/TR/wai-aria-practices-1.1/examples/slider/multithumb-slider.html)
